### DISCUSSION TITLE
Device in Device bays with image

### DISCUSSION THREAD

**Discussion by wannagofast83** [netbox-community/netbox#12078] [Ideas]
We have a horizontal rack in some of our racks that hold different devices. I setup a device that is the parent with the device bays that goes on the rails(child).

Rail mount that holds a DC power supply that sends power over a cable via copper to another cabinet. I like the idea of the image of the child over the parent. Is there a way to do so?

**Comment by candlerb** [comment]
No. The only image you get is of the rackable (parent) device.  You could have multiple variations of this device type with different images though.
